src/components/MenuItem.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+
+export class MenuItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      bread: "Standard",
+      isHot: false,
+      hotPrice: 0.2,
+      total: this.props.price
+    };
+  }
+
+  addBread(event) {
+    let breadName = this.props.breads[event.target.value].name;
+    let breadPrice = this.props.breads[event.target.value].price;
+    let total = this.props.price + breadPrice;
+
+    this.setState({
+      bread: breadName,
+      total: !this.state.isHot ? total : total + this.state.hotPrice
+    });
+  }
+
+  addHot() {
+    const hotPrice = this.state.hotPrice;
+    let isHot = this.state.isHot ? false : true;
+
+    this.setState({
+      isHot: isHot,
+      total: isHot ? this.state.total + hotPrice : this.state.total - hotPrice
```

Comment:
This is a bit confusing. Why does `hotPrice` need deducting? Also can do the calculations in a variable above to make the `setState` statement cleaner too

Reply:
Ah sure, this is the issue with handling the price on the `MenuItem` rather than at the order side. You have to adjust the price locally rather than at the end

Reply:
sure that makes sense man - need a refactor of this then (in the future)